zulip/zulip/__init__.py

```diff
@@ -1554,6 +1554,53 @@ def set_typing_status(self, request: Dict[str, Any]) -> Dict[str, Any]:
             request=request
         )
 
+    def move_topic(
+        self,
+        stream: str,
+        new_stream: str,
+        topic: str,
+        new_topic: Optional[str] = None
+    ) -> Dict[str, Any]:
+        """Move a topic from ``stream`` to ``new_stream``
+
+        The topic will be renamed if ``new_topic`` is provided
+
+        Example usage::
+
+            >>> client.move_topic('stream_a', 'stream_b', 'my_topic')
+            {'result': 'success', 'msg': ''}
```

Comment:
This is over-indented, I think.

Reply:
I've changed the docstring to match the exiting style